governator-legacy/src/main/java/com/netflix/governator/guice/InternalAutoBindModule.java

```diff
@@ -184,8 +186,14 @@ else if ( Module.class.isAssignableFrom(clazz) )
 
     private void bindAutoBindSingleton(AutoBindSingleton annotation, Class<?> clazz)
     {
-        LOG.info("Installing @AutoBindSingleton " + clazz.getName());
+        LOG.info("Installing @AutoBindSingleton '{}'", clazz.getName());
+        LOG.info("***** @AutoBindSingleton for '{}' is deprecated soon.\nPlease use a Guice module with bind({}.class).asEagerSingleton(); instead", 
```

Comment:
This is definitely a better message than on the deprecated annotation, but we may still just want the wiki page.